generators/client/templates/react/src/main/webapp/app/modules/account/password/password-strength-bar.tsx

```diff
@@ -0,0 +1,79 @@
+import './password-strength-bar.<%= styleSheetExt %>';
+
+import * as React from 'react';
+import { Translate } from 'react-jhipster';
+
+export interface IPasswordStrengthBarProps {
+  password: string;
+}
+
+export default class PasswordStrengthBar extends React.Component<IPasswordStrengthBarProps> {
```

Comment:
this can be a stateless component like below
```
export const PasswordStrengthBar = ({ password }: IPasswordStrengthBarProps) => {
  const colors = [...];
  const measureStrength = () => {};
  ...
  const strength = getColor(measureStrength(password));
  const points = getPoints(strength);
  return (
    <div id="strength">
    ...
    </div>
  );
}
export default PasswordStrengthBar;
```